test/quantization/test_quantized_op.py

```diff
@@ -2128,6 +2130,129 @@ def test_qlinear(self, batch_size, input_channels, output_channels,
         self.assertEqual(Y_fp32, Y_fp32_ref,
                          msg="torch.ops.quantized.linear_dynamic results are off")
 
+class TestDynamicQuantizedRNNOp(TestCase):
+    """Tests the correctness of the dynamic quantized lstm/gru."""
+    @given(
+        num_batches=st.integers(1, 4),
+        input_size=st.integers(16, 32),
+        hidden_size=st.integers(4, 8),
+        num_directions=st.integers(1, 2),
+        per_channel_quant=st.booleans())
+    @override_qengines
+    def test_qlstm(self, num_batches, input_size, hidden_size,
+                   num_directions, per_channel_quant):
+
+        def _get_rnn_inputs(seq_len, num_batches, input_size, hidden_size, num_directions):
+            # For Input (seq_len, batch, input_size)
+            X = torch.randn(seq_len, num_batches, input_size)
+            # TODO: Change to reduce_range=True once support is enabled
```

Comment:
This is supported now

Reply:
I have it enabled in the next PR in the stack, will move the change to this PR itself.